doc/classes/Skeleton3D.xml

```diff
@@ -31,6 +31,17 @@
 				[i]Deprecated soon.[/i]
 			</description>
 		</method>
+		<method name="bone_transform_to_world_transform">
+			<return type="Transform">
+			</return>
+			<argument index="0" name="bone_transform" type="Transform">
+			</argument>
+			<description>
+				Takes the given bone pose/transform and converts it to a world transform, relative to the [Skeleton3D] node.
+				
```

Comment:
Spaces are different in documentation. I think this causes an extra space. Please check documentation text.

Reply:
This is an outdated bit from #39126. Once said PR is pulled in, I'll rebase this PR so that commit is no longer there. This should make the documentation the same as #39126.

I do need to document what I have added/changed via this PR though...